msamp/nn/__init__.py

```diff
@@ -9,6 +9,8 @@
 from msamp.nn.linear import FP8Linear, LinearReplacer
 from msamp.nn import functional
 from msamp.nn.clip_grad import clip_grad_norm_
+from msamp.nn import distributed
```

Comment:
Can we put attributes from same module together?
`from msamp.nn import functional, distributed`

Reply:
Sure. Fixed.